astronomer/providers/amazon/aws/example_dags/example_batch.py

```diff
@@ -137,6 +138,27 @@ def get_job_queue_status() -> str:
         return "DELETED"
 
 
+def list_jobs_func() -> str:
+    """Get the list of aws batch jobs for the particular Job Name"""
+    import boto3
+    from botocore.exceptions import ClientError
+
+    client = boto3.client("batch")
+
+    try:
+        response = client.list_jobs(jobQueue=JOB_QUEUE, filters=[{"name": "JOB_NAME", "values": [JOB_NAME]}])
+    except ClientError:
+        response = {}
+
+    logging.info("%s", response)
+    if response.get("jobSummaryList"):
+        res = response.get("jobSummaryList")[0]
+        job_id: str = res.get("jobId")
+    else:
+        job_id = "failed to fetch job id"
```

Comment:
Would it mean here that Job will relevant name was not found or does not exist? In that case I believe we should raise an error here upfront and skip further processing to retrieve and sense the status of the job; implicitly it would mark the task status to failed.

Reply:
done